api/GeolocationPositionError.json

```diff
@@ -67,7 +67,7 @@
           },
           "safari_ios": {
             "alternative_name": "PositionError",
-            "version_added": true
+            "version_added": "5"
```

Comment:
Safari 5 should map to 4.2 using the mapping in https://github.com/mdn/browser-compat-data/pull/11299.

Can you (somehow) fix the mirroring script to match https://github.com/mdn/browser-compat-data/pull/11299 and update this PR?

Reply:
I've updated this PR with a manual fix (`GeolocationPositionError` was the only affected file in this PR).  I'm hoping to get the other mirroring script changes landed first before opening more to prevent merge conflicts (particularly since some of my own PRs change the logic flow of the script a bit).